cmake/sysbuild/fast_pair/hex.cmake

```diff
@@ -143,6 +169,25 @@ function(fast_pair_hex_dts)
   )
 endfunction()
 
+function(fast_pair_device_model_warning)
+  # Maintains the list of Nordic owned Fast Pair device models. These models are automatically
+  # supplied to the Fast Pair projects that are maintained in the nRF Connect SDK.
+  if(SB_CONFIG_BT_FAST_PAIR_MODEL_ID STREQUAL "0x2A410B" OR
+     SB_CONFIG_BT_FAST_PAIR_MODEL_ID STREQUAL "0x4A436B" OR
+     SB_CONFIG_BT_FAST_PAIR_MODEL_ID STREQUAL "0x52FF02" OR
```

Comment:
Would it work? `SB_CONFIG_BT_FAST_PAIR_MODEL_ID` is no longer a string

Reply:
it works (the hex type is probably converted into string before the STREQUAL operation), but let's avoid unnecessary conversions